portal/src/main/webapp/config_service.jsp

```diff
@@ -114,6 +114,7 @@
             "oncoprint.custom_driver_annotation.tiers.default",
             "ensembl.transcript_url",
             "enable_persistent_cache",
+            "enable_request_body_compression",
```

Comment:
I guess this reports the setting of the property to the frontend perhaps?  That way the front end will know whether it can use compression or not. Ok

Reply:
Exactly.